pom.xml

```diff
@@ -0,0 +1,21 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.redhat.developer</groupId>
+  <artifactId>event-connect</artifactId>
+  <version>0.0.1</version>
```

Comment:
Please use `0.0.1-SNAPSHOT` as version here and in all other poms. Fixed version becomes a problem as soon as we will configure nightly maven deploy